forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -5821,9 +5823,15 @@ public final int getKeywordMagnitude(final Keyword k, CardState state) {
             if (StringUtils.isNumeric(s)) {
                count += Integer.parseInt(s);
             } else {
-                String svar = StringUtils.join(parse);
-                if (state.hasSVar(svar)) {
-                    count += AbilityUtils.calculateAmount(this, state.getSVar(svar), null);
+                StaticAbility st = inst.getStatic();
+                // TODO make keywordinterface inherit from CardTrait somehow, or invent new interface
+                if (st != null && st.hasSVar(s)) {
+                    count += AbilityUtils.calculateAmount(this, st.getSVar(s), null);
+                } else {
+                    String svar = StringUtils.join(parse);
+                    if (state.hasSVar(svar)) {
+                        count += AbilityUtils.calculateAmount(this, state.getSVar(svar), null);
+                    }
```

Comment:
@tool4ever can we safely remove this part?

Reply:
fallback might still be needed in some cases when "fully" intrinsic e. g. _Petrified Wood-Kin_ with "Bloodthirst X" (though this one might need to be unhardcoded first)

Reply:
The difference there is that Bushido Xis gained via Static Ability, while Bloodthirst X is a direct keyword.


Bloodthirst X is hardcoded because it is special in the rules:

> 702.54b “Bloodthirst X” is a special form of bloodthirst. “Bloodthirst X” means “This permanent enters with X +1/+1 counters on it, where X is the total damage your opponents have been dealt this turn.”

and also doesn't use getKeywordMagnitude
* This one is only used for Toxic and Bushido
* And only Bushido is gained via special Static